packages/sanity/src/core/studio/components/navbar/resources/helper-functions/helpResources.ts

```diff
@@ -0,0 +1,27 @@
+import type {Observable} from 'rxjs'
+import {map} from 'rxjs/operators'
+import {SanityClient} from '@sanity/client'
+import {SANITY_VERSION} from '../../../../../version'
+import {ResourcesResponse} from './types'
+
+/**
+ * @internal Not a stable API yet
```

Comment:
`@hidden` tag is also needed to make sure it gets hidden from the docs

```suggestion
 * @internal Not a stable API yet
 * @hidden
```